tests/Backend/Workspaces/MetadataFromRedshiftWorkspaceTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\Workspaces;
+
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\Backend\Workspaces\WorkspacesTestCase;
+
+class MetadataFromRedshiftWorkspaceTest extends WorkspacesTestCase
+{
+    public function testCreateTableFromWorkspace()
+    {
+        // create workspace and source table in workspace
+        $workspaces = new Workspaces($this->_client);
+        $workspace = $workspaces->createWorkspace(["backend" => "redshift"]);
+        $connection = $workspace['connection'];
+        $db = $this->getDbConnection($connection);
+        $db->query("create table \"test.Languages3\" (
+                    \"Id\" varchar not null,
+                    \"Name\" varchar null
+                );");
+        // create table from workspace
+        $tableId = $this->_client->createTableAsyncDirect($this->getTestBucketId(self::STAGE_IN), array(
+            'name' => 'languages3',
+            'dataWorkspaceId' => $workspace['id'],
+            'dataTableName' => 'test.Languages3',
+        ));
+        $expectedIdMetadata = [
+            'KBC.datatype.type' => 'varchar',
+            'KBC.datatype.nullable' => '',
+            'KBC.datatype.basetype' => 'STRING',
+            'KBC.datatype.length' => '256',
+        ];
+        $expectedNameMetadata = [
+            'KBC.datatype.type' => 'varchar',
+            'KBC.datatype.nullable' => '1',
+            'KBC.datatype.basetype' => 'STRING',
+            'KBC.datatype.length' => '256',
+        ];
+        // check that the new table has the correct metadata
+        $table = $this->_client->getTable($tableId);
+        $this->assertEquals([], $table['metadata']);
+        $this->assertArrayHasKey('id', $table['columnMetadata']);
```

Comment:
Here I got this:
![Snímek z 2019-05-20 16-29-36](https://user-images.githubusercontent.com/5272798/58029301-91631000-7b1c-11e9-8cba-371d27f2569e.png)
And test failed. Maybe something wrong on my side ?

Reply:
Solved ... missing project feature enabled